lib/alpha_taxonomy/taxon_link_deleter.rb

```diff
@@ -0,0 +1,27 @@
+module AlphaTaxonomy
+  class TaxonLinkDeleter
+    def initialize(logger: Logger.new(STDOUT))
+      @log = logger
+    end
+
+    def run!
+      base_paths = ENV.fetch("TAXON_CLEANUP").split(",")
+
+      base_paths.each do |base_path|
+        @log.info "Clearing links for #{base_path}"
+
+        linked_items = Services.live_content_store.incoming_links!(
+          base_path, types: ['alpha_taxons']
+        )['alpha_taxons']
+
+        linked_items.each do |linked_item|
```

Comment:
I'm pretty sure it's an empty array in that case. Will cast this anyway, it'll be a bit more robust at least.